.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+# Changes
+<!-- Replace this line with a description of the changes you made -->
+
+# Testing
+<!-- Replace this line with a quick way of testing your changes work as expected -->
```

Comment:
Donc ça c'est un document que tu mets à chaque PR ? Mais genre tu les stockes où ? Parce que si chacun fait une PR ça va faire des tonnes et des tonnes de fichier non ?

Reply:
No, this is only the pull request template. It'll show up as default content when you create a PR on the repo.